### ISSUE TITLE
dashboard elements

### ISSUE THREAD

**Issue by grandridgelabs** [grandridgelabs/fx-saas#19]
- [ ] all rules for user (show 10 by default)
- [ ] show last 10 backtests

**Comment by Joker9090** [comment]
New Pr with this features mocked -> https://github.com/grandridgelabs/fx-saas/pull/20

**Comment by grandridgelabs** [comment]
looks good, can we put them side by side?  also in the backtest table, can we replace rule id with rule name.

these are low priority changes

**Comment by Joker9090** [comment]
Of course! but I recommend you to only display the tables side a side when the viewport (browser windows) is bigger than 1200px.

This way make the view little more responsive.
The tables have a lot of content. Otherwise there will be a lot of tight info.

What do you think?

**Comment by Joker9090** [comment]
![table resp](https://user-images.githubusercontent.com/8048431/52737841-0e299f80-2fac-11e9-8df6-2d2338ba85b5.gif)

**Comment by grandridgelabs** [comment]
perfect, yes and good call on device

**Comment by grandridgelabs** [comment]
we want to be able to kick off backtests from dashboard, and id like it to be a popup div if possible, or some kind of expansion i.e. hidden -> visible to select pairs, date range etc.  perhaps we can use same control on both dashboard and /rules as we will still want to kick off backtest from /rules
